### ISSUE TITLE
is cordova-sqlite-evcore-extbuild-free a drop-in replacement for cordova-sqlite-storage?

### ISSUE THREAD

**Issue by qpwr** [brodybits/ask-me-anything#34]
Hey Brody,

after my app is available for some time now, I'm starting to have users report memory issues on Android now and then. I understand that cordova-sqlite-evcore-extbuild-free has a better memory handling than cordova-sqlite-storage, but I couldn't find any information on whether the first is a drop-in replacement for the latter or if it comes with changes to the API or other things. Could you shed some light on this? Thanks a lot!

Marc

**Comment by brody2consult** [comment]
Yes the evcore plugin version can be used as a drop-in replacement, with GPL v3 or commercial licensing options. For commercial license inquiries: <sales@storesafe.io>
